src/streaming/controllers/ScheduleController.js

```diff
@@ -199,17 +199,17 @@ function ScheduleController(config) {
 
         validateExecutedFragmentRequest();
 
-        const isReplacement = replaceRequestArray.length > 0;
-        const readyToLoad = bufferLevelRule.execute(streamProcessor, type, streamController.isVideoTrackPresent());
-        const topQualityChanged = hasTopQualityChanged(currentRepresentationInfo.mediaInfo.type, streamProcessor.getStreamInfo().id);
+        let isReplacement, topQualityChanged, readyToLoad;
```

Comment:
This i hard to read.  Are you assigning the var in the if statement.  We should keep code standards.  We do not list vars in one line.  Use const if not reassigned.  Used keyword per var being assigned for break point access etc.